### ISSUE TITLE
Mode C

### ISSUE THREAD

**Issue by AndyTWF** [VATSIM-UK/uk-controller-plugin#459]
It stops stands from being sent to vStrips, it stops initial altitudes being assigned... We need a way to handle it.

**Comment by kristiankunc** [comment]
Not sure how much is this still relevant.
cc @hazzas-99

**Comment by hazzas-99** [comment]
I don't think it works perfectly still @kristiankunc. Certainly when I'm on London using Mode C and I try to recycle the initial climb on an a/c that it isn't set for, the plugin doesn't seem to be able to set the initial climb.
